TROUBLESHOOTING.md

```diff
@@ -53,7 +55,20 @@
   - GCP:
 
   - Azure:
-   
+
+<hr style="height:1px;border:none;color:#333;">
+
+### Restic - NFS volumes and `root_squash`:
+
+If using NFS volumes while `root_squash` is enabled, Restic will be mapped to 
+`nfsnobody` and not have the proper permissions to perform a backup/restore. 
+
+#### To solve this issue:
+  - Use supplemental groups, and apply this same supplemental group to the Restic
```

Comment:
Please include a link/reference to the `ResticSupplementalGroups` config value in our CR spec. This is how the restic daemonset will have these gids applied to the daemonset. Other than that looks good